### PR TITLE
Fixes #111: Fix caching issues while with tests

### LINKED ISSUE #111: Caching issues
* Expires value needs to be a RFC 1123 date http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.21
* maxAge needs to be defined in seconds. It's used by express-session to set cookie's max-age value (https://www.ietf.org/rfc/rfc2109.txt).
* Modern browsers prefer Cache-Control header over Expires header.
* Pragma is obsolete: http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.32

### PR DESCRIPTION
**Pull request by santiagobasulto**

### CODE REVIEW

File: app/utils/caching.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+var _str = require('underscore.string');
+var path = require('path');
+var helmet = require('helmet');
+
+_.mixin(_str.exports());
+
+exports = module.exports = function(IoC) {
+  return {
+    getCachingHeadersFromSettings: function(settings){
+      var maxAge = parseInt(settings.staticServer.maxAge);
```
Comment (BrunoBernardino) [inline app/utils/caching.js]: Please add the `parseInt()`'s second argument. `var maxAge = parseInt(settings.staticServer.maxAge, 10);`

File: templates/boot/config.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ var publicDir = path.join(assetsDir, 'public');
 var templatesDir = path.join(assetsDir, 'emails');
 var viewsDir = path.join(appDir, 'views');
 
-var maxAge = 24 * 60 * 60 * 1000;
+var maxAge = 365 * 24 * 60 * 60;  // One year in seconds
```
Comment (BrunoBernardino) [inline templates/boot/config.js]: Are we sure that for `session` it's in seconds? I had the feeling [it should be in milliseconds](https://github.com/expressjs/session#cookiemaxage).

File: app/utils/caching.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      var headers = [];
+
+      if(maxAge){
```
Comment (BrunoBernardino) [inline app/utils/caching.js]: Might be best to rewrite this as returning the headers if there's no maxAge, to confirm with the coding style and don't indent if not necessary.

**Comment by BrunoBernardino** [comment]
Great PR! Thanks.

Apart from my comments, to improve the code to match the style guide:

1. You're using double quotes (`"`) when single `'` can be used
2. You're doing a lot of `){` when there should be a space, like `) {` (in ifs, functions, etc.)

**Comment by santiagobasulto** [comment]
This reminds me the days of Java with the different conventions about leaving the bracket up or down. People went crazy about that. So funny.

**Pull request closed without merging**